src/main/java/edu/harvard/iq/dataverse/api/Dataverses.java

```diff
@@ -743,6 +749,52 @@ public Response updateGroup(ExplicitGroupDTO groupDto,
                 new UpdateExplicitGroupCommand(req,
                         groupDto.apply(findExplicitGroupOrDie(findDataverseOrDie(dvIdtf), req, grpAliasInOwner)))))));
     }
+    
+    @PUT
+    @Path("{identifier}/defaultContributorRole/{roleAlias}")
+    public Response updateDefaultContributorRole(
+            @PathParam("identifier") String dvIdtf,
+            @PathParam("roleAlias") String roleAlias) {
+
+        DataverseRole defaultRole;
+        
+        if (roleAlias.equals("none")) {
```

Comment:
Ok, it's more clear to me now that "none" is a special keyword, not something that appears in the UI. Right? Can we make "none" into a constant? Otherwise we have "none" sprinkled in various places in the code.